lib/elixir/lib/calendar/date.ex

```diff
@@ -496,7 +496,7 @@ defmodule Date do
       ~D[2000-01-03]
 
   """
-  @spec add(Calendar.date, integer()) :: t
```

Comment:
Hold on this change because the formatter will format `Calendar.date()` (because it's a remote call), so for consistency we might want to go with `integer()` as well (local calls have parens in code as well). We might change it but I think we should hold off on this change for now.